frontend/web/components/PanelSearch.js

```diff
@@ -329,7 +329,9 @@ const PanelSearch = class extends Component {
             />
           )}
           {this.props.renderFooter && (
-            <div className='panel-footer'>{this.props.renderFooter()}</div>
+            <footer className='panel-footer'>
```

Comment:
Border radius was incorrect and removed the border

<img width="1214" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/8608314/52164a8b-69f2-4ded-a5bb-398f5c24c9d5">



<img width="1187" alt="image" src="https://github.com/Flagsmith/flagsmith/assets/8608314/bafdb79f-b2ce-4234-a6a4-31b1154afeab">